lib/KBaseReport/utils/template_utils.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+from template import Template
+
+""" Utility for rendering a report from a template """
```

Comment:
Minor, but I would put this docstring at the top of the file. That way it will show up if you do `help(module)`